Quick-Start [DA]/Sample-Ext.md

```diff
@@ -1 +1,259 @@
+# Extending Postgres with `pg_sample_ext`
 
+In this quick start guide, we will explore the process of extending PostgreSQL using the `pg_sample_ext` extension. `pg_sample_ext` is a sample extension that gradually introduces various extension features which include
+- Functions
+- Data Types
+  - `CREATE TYPE`
+  - `CREATE DOMAIN`
+- Operators
+
+By following this guide, you will learn how to create, install, and upgrade the `pg_sample_ext` extension while understanding the different ways in which Postgres can be extended. Each section will introduce a specific feature, providing hands-on examples and explanations to help you grasp the concepts and apply them to your own extensions.
+
+Throughout this quick start, we will start with the basics, such as creating a function then we will progress to more advanced topics like defining custom data types and domains, implementing operators for custom types, and exploring advanced indexing and table access methods.
+
+By the end of this guide, you will have a solid understanding of how to extend PostgreSQL using the `pg_sample_ext` extension and be equipped with the knowledge to create your own powerful extensions, tailored to meet your specific needs.
+
+## Function: 
```

Comment:
This is super brief, which is great! But I suggest linking to the `my_extension` tutorial for people who need more detail to get started.

Reply:
sure.